Python/Watermark_Images/watermark_logo.py

```diff
@@ -0,0 +1,39 @@
+import cv2
+import numpy as np
+
+#importing the main image
+pImg = '/content/food.png' #path to main image
+image = cv2.imread(pImg)
+oH,oW = image.shape[:2]
+image = np.dstack([image, np.ones((oH,oW), dtype="uint8") * 255])
+
+
+#importing the logo image
+pLogo = '/content/tastelogo.png' #path to logo

```

Comment:
Is it possible to pass the logo file path dynamically?

Reply:
Done